http/src/request/channel/message/update_message.rs

```diff
@@ -161,6 +171,30 @@ impl<'a> UpdateMessage<'a> {
         Ok(self)
     }
 
+    /// Supress the embeds in the message.
+    pub fn supress_embeds(mut self, supress: bool) -> Self {
+        let mut flags = self.fields.flags.unwrap_or_else(MessageFlags::empty);
+
+        if supress {
+            flags |= MessageFlags::SUPPRESS_EMBEDS;
+        } else {
+            flags &= !MessageFlags::SUPPRESS_EMBEDS;
+        }
+        self.fields.flags.replace(flags);
+
+        self
+    }
+
+    /// Set the allowed mentions in the message, use the [`build_solo`]
```

Comment:
The header of this documentation should just be "Set the allowed mentions in the message." and then the reference to build_solo on another line to keep in style with the rest of our documentation